MoltenVK/MoltenVK/Commands/MVKCommandBuffer.h

```diff
@@ -494,6 +495,7 @@ class MVKCommandEncoder : public MVKBaseDeviceObject {
 	VkRect2D _renderArea;
     MVKActivatedQueries* _pActivatedQueries;
 	MVKSmallVector<VkClearValue, kMVKDefaultAttachmentCount> _clearValues;
+	MVKSmallVector<MVKImageView*, kMVKDefaultAttachmentCount> _imagelessAttachments;
```

Comment:
I feel like there ought to be a better way to handle this. The imageless model, with attachments provided at render pass begin time, corresponds better to the Metal model. Maybe we should see about not keeping the framebuffer object around and just using a vector of attachments, since the purpose of the framebuffer object is to be a container for attachments.